packages/@tinacms/graphql/src/mdx/stringify.ts

```diff
@@ -439,6 +439,64 @@ ${out}
       }
 
     case 'text':
```

Comment:
It might be nice to refactor this with "wrapping" or "Decorator" style of code. It just might make it a bit more maintainable.

For example.
```ts
case 'text':
      let returnNode: Content = { type: 'text', value: node.text || '' }

      if (node?.bold) {
        returnNode = { type: 'strong', children: [returnNode] }
      }
      if (node?.italic) {
        returnNode = {
          type: 'emphasis',
          children: [returnNode],
        }
      }
      return returnNode
```

Should do the trick.